### DISCUSSION TITLE
Swagger with api.yml config

### DISCUSSION THREAD

**Discussion by OctavioBernalGH** [DarkaOnLine/L5-Swagger#572] [Q&A]
I have installed swagger in Laravel, I want it to read the api.yml file hosted in Storage/api-docs, there I have the necessary endpoints, how do I do it?

**Comment by Rikj000** [comment]
First publish L5-Swagger's config as described in the wiki:
https://github.com/DarkaOnLine/L5-Swagger/wiki/Installation-&-Configuration#for-versions--6

Then under `config/l5-swagger.php` configure:
- `documentations.routes`
- `documentations.paths.docs_json` or `documentations.paths.docs_yaml`
- `documentations.paths.format_to_use_for_docs`
